addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -400,26 +400,34 @@
 				<bordersize>20</bordersize>
 				<include>OpenClose_Left</include>
 			</control>
-			<control type="grouplist">
-				<left>420</left>
-				<top>10</top>
-				<right>20</right>
-				<height>300</height>
-				<align>left</align>
-				<orientation>vertical</orientation>
-				<itemgap>10</itemgap>
-				<control type="textbox">
-					<height>250</height>
-					<label fallback="10005">$INFO[VideoPlayer.Tagline,[B],[/B][CR]]$INFO[VideoPlayer.Plot]</label>
-					<align>left</align>
-					<autoscroll delay="5000" repeat="7500" time="5000"></autoscroll>
-				</control>
-				<control type="label">
-					<height>50</height>
-					<label>$VAR[OSDNextLabelVar]</label>
-					<visible>Integer.IsGreater(Playlist.Length(video),1)</visible>
-				</control>
-			</control>
-		</control>
-	</controls>
+      <control type="grouplist">
+        <left>420</left>
+        <top>10</top>
+        <right>20</right>
+        <height>300</height>
+        <align>left</align>
+        <orientation>vertical</orientation>
+        <itemgap>10</itemgap>
+        <control type="textbox">
+          <height>295</height>
```

Comment:
Height needs reducing by 1px otherwise you can see the top of the next line when at 1280x720.